### DISCUSSION TITLE
error on V2RayN

### DISCUSSION THREAD

**Discussion by nicolauhk** [apernet/hysteria#924] [Q&A]
Is any expert can help me why I get follow error in Windows V2RayN

I can access internet by I got error

ERROR [977136803 7.73s] inbound/http[http]: process connection from 127.0.0.1:50505: download: write tcp 127.0.0.1:10809->127.0.0.1:50505: wsasend: An established connection was aborted by the software in your host machine.

**Comment by meower1** [comment]
I suggest you use [nekoray](https://github.com/MatsuriDayo/nekoray) for hysteria and choose singbox as the default core upon startup. it works hassle free with hysteria and supports a wider range of protocols
